schema.jsonld

```diff
@@ -0,0 +1,47 @@
+{
+  "@context":{
+    
+    "schema":"http://www.schema.org/",
+    "codemeta":"https://codemeta.github.io/terms/",
+    
+    "Organization":{"@id":"schema:Organization"},
+    "Person":{"@id":"schema:Person"},
```

Comment:
What is Person? authors?

Reply:
Person is a type in schema allows to describe person information
authors can by define by Person type. My context is almost the same as in meta data code.

Reply:
Yes. But here it is a field, not a type, isn't t?